### PR TITLE
dns/bind: revamp logging

### PR DESCRIPTION
**Pull request by sestary**
This PR revamps the logging pages:

- Combines the three pages into one page with multiple tabs.
- Adds a log format handler to parse out the individual fields.

Result:

Screenshot 2022-12-18 at 9 32 52 PM

### CODE REVIEW

File: dns/bind/pkg-descr [hunk 1]
```diff
@@ -9,6 +9,10 @@ WWW: https://www.isc.org
 Plugin Changelog
 ================
 
+1.26
```
Comment (fichtner) [inline dns/bind/pkg-descr:12]: 1.26 hasn't been released so you can move these additions to 1.25

File: dns/bind/Makefile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		bind
-PLUGIN_VERSION=		1.25
+PLUGIN_VERSION=		1.26
```
Comment (fichtner) [inline dns/bind/Makefile:2]:
```suggestion
PLUGIN_VERSION=		1.25
```

File: dns/bind/pkg-descr (hunk 1 above)
Comment (sestary) [reply dns/bind/pkg-descr]: Thanks have done that.

File: dns/bind/Makefile (hunk 2 above)
Comment (sestary) [reply dns/bind/Makefile]: Same here

**Comment by fichtner** [comment]
Merged, thanks a lot for working on this! ❤️

**Pull request merged**